featuretools/utils/utils_info.py

```diff
@@ -17,15 +16,13 @@
     "dask",
     "distributed",
     "psutil",
-    "Click",
     "pip",
     "setuptools",
 ]
 
 
 def show_info():
-    res = subprocess.run(["featuretools", "info"], stdout=subprocess.PIPE)
-    print(res.stdout.decode("utf-8"))
+    print_info()
```

Comment:
I think we should put the contents of `print_info` here and delete `print_info`, it's not used anywhere else

Reply:
Ah gotcha, fixed.